airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/avro/AvroRecordFactory.kt

```diff
@@ -22,6 +25,29 @@ class AvroRecordFactory(private val schema: Schema?, private val converter: Json
     companion object {
         private val MAPPER: ObjectMapper = MoreMappers.initMapper()
         private val WRITER: ObjectWriter = MAPPER.writer()
+
+        fun createV1JsonToAvroConverter(): JsonAvroConverter {
```

Comment:
should we mark this as deprecated?

Reply:
Possibly. It's still in use by GCS at least, but should go away completely when that's done.